plugins/eddn.py

```diff
@@ -773,6 +1042,31 @@ def plugin_stop() -> None:
     logger.debug('Done.')
 
 
+# Recursively filter '*_Localised' keys from dict
+def filter_localised(d: Mapping[str, Any]) -> OrderedDictT[str, Any]:
+    """
+    Remove any dict keys with names ending `_Localised` from a dict.
+
+    :param d: Dict to filter keys of.
+    :return: The filtered dict.
+    """
+    filtered: OrderedDictT[str, Any] = OrderedDict()
```

Comment:
do we truly need this to be an OrderedDict? I dont remember

Reply:
I believe you left it that way from original pre-python3 code (or maybe from when I ported to python3).

In theory it doesn't need to be explicitly Ordered now, but I'm not changing it in *this* PR.